README.md

```diff
@@ -60,7 +60,8 @@ See the Examples section below for more details, or the [documentation](https://
 * [RWC Jazz](https://staff.aist.go.jp/m.goto/RWC-MDB/rwc-mdb-j.html)
 * [RWC Popular](https://staff.aist.go.jp/m.goto/RWC-MDB/rwc-mdb-p.html)
 * [Salami](https://github.com/bmcfee/salami-data-public/tree/hierarchy-corrections)
-* [Saraga](https://mtg.github.io/saraga/)
+* [Saraga Carnatic](https://mtg.github.io/saraga/)
+* [Saraga Hindustani](https://mtg.github.io/saraga/)
```

Comment:
Just checking: is the exact same `url` for both datasets I guess?

Reply:
Yes! Should I write it in a single line like:
`[Saraga Carnatic & Hindustani](https://mtg.github.io/saraga/)`

Reply:
No it's ok like this! Just checking, thanks